config/initializers/rails_admin.rb

```diff
@@ -2,6 +2,8 @@
 require 'rails_admin/config/actions/unpublish'
 
 RailsAdmin.config do |config|
+  config.main_app_name = ["Europeana Collections"]
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.